README.md

```diff
@@ -19,45 +19,20 @@
     
     **Note:** Not all `git` versions can recognize `--remote-submodules`. If you are cloning this git simply just to compile, you can omit this and you'll be fine. Most IDEs and the MinGW library offered in step 1 should be able to recognize `--remote-submodules`
 
-5. `cd` into the build folder:
+5. Execute the build script:
 
     ```bash
-    cd SSVOpenHexagon/build
+    sh ./build.sh
```

Comment:
```suggestion
    bash ./build.sh
```

Reply:
Switching from `sh` to `bash` changes the build script to build on `linux-gnu` as opposed to `msys` for MinGW. Does switching to `bash` matter in this case?

Reply:
Probably not. Let's keep it as is and see how it goes.